programs/programs/tx/fpp/tests/test_fpp.py

```
@@ -31,46 +39,53 @@ def make_member(age=30, medicaid=False, emergency_medicaid=False, employer=False
+      - `child_support_paid` — screen-level `childSupport` expense.
+
+    The §4140 adjunctive bypass reads enrollment as a real screen exposes it: SNAP/WIC from
+    the CurrentBenefit join table via has_benefit(), CHIP from per-member insurance via
+    has_insurance_types(("chp",)) — not the legacy has_snap/has_wic/has_chp columns.
     """
     mock_program = Mock()
     mock_program.year.get_limit.return_value = fpl_limit
 
     benefits = set(current_benefits or [])
 
+    def _gross(freq, types, exclude=[]):
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)